server/validators/phase.ts

```diff
@@ -1,16 +1,16 @@
+import * as yup from 'yup';
 import { validISODateString } from './helpers';
 
-// eslint-disable-next-line @typescript-eslint/no-var-requires
-const yup = require('yup');
-
 const dateValidation = {
   start_date: yup
     .string()
     .required('Start date is required')
     .test('is-date', 'Not a valid date', validISODateString),
   end_date: yup
     .string()
-    .when('start_date', (startDate, schema) => {
+    // See https://github.com/jquense/yup/issues/1901 for why this is needed
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    .when('start_date', (startDate: any, schema) => {
```

Comment:
Does this need to be `any`?

Reply:
Yes. There seems to be a bug in `yup`'s typing - forcefully making this `any` prevents TS from complaining about stuff that isn't actually an issue. Hopefully once `yup` gets patched, we can pull all of this out.